object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -350,340 +304,31 @@ public ObjectConstructionTypeAnnotator(AnnotatedTypeFactory atypeFactory) {
-   */
-  private Set<String> getAutoValueBuilderSetterMethodNames(Element builderElement) {
-    return getAllAbstractMethods(builderElement).stream()
-        .filter(e -> isAutoValueBuilderSetter(e, builderElement))
-        .map(e -> e.getSimpleName().toString())
-        .collect(Collectors.toSet());
-  }
-
-  /**
-   * For a toBuilder routine, we know that the returned Builder effectively has had all the required
-   * setters invoked. Add a CalledMethods annotation capturing this fact.
-   *
-   * @param t type of toBuilder method
-   * @param classElement enclosing class
-   * @param b whether this toBuilder was generated by Lombok or AutoValue
-   */
-  private void handleToBuilder(
-      AnnotatedTypeMirror.AnnotatedExecutableType t, Element classElement, BuilderKind b) {
-    AnnotatedTypeMirror returnType = t.getReturnType();
-    handleToBuilderType(returnType, returnType.getUnderlyingType(), classElement, b);
-  }
-
-  /**
-   * Update a particular type associated with a toBuilder with the relevant CalledMethods
-   * annotation. This can be the return type of toBuilder or the corresponding generated "copy"
-   * constructor
-   *
-   * @param type type to update
-   * @param builderType type of abstract @AutoValue.Builder class
-   * @param classElement corresponding AutoValue class
-   * @param b whether the builder was generated by AutoValue or Lombok
-   */
-  private void handleToBuilderType(
-      AnnotatedTypeMirror type, TypeMirror builderType, Element classElement, BuilderKind b) {
-    Element builderElement = TypesUtils.getTypeElement(builderType);
-    Set<String> avBuilderSetterNames = getAutoValueBuilderSetterMethodNames(builderElement);
-    List<String> requiredProperties = getRequiredProperties(classElement, avBuilderSetterNames, b);
-    AnnotationMirror calledMethodsAnno =
-        createCalledMethodsForProperties(requiredProperties, avBuilderSetterNames, b);
-    type.replaceAnnotation(calledMethodsAnno);
-  }
-
-  /**
-   * computes the required properties of a builder class
-   *
-   * @param builderElement the class whose builder is to be checked
-   * @param avBuilderSetterNames
-   * @param builderKind the framework by which the builder will be generated
-   * @return a list of required property names
-   */
-  private List<String> getRequiredProperties(
-      final Element builderElement,
-      Set<String> avBuilderSetterNames,
-      final BuilderKind builderKind) {
-    switch (builderKind) {
-      case AUTO_VALUE:
-        return getAutoValueRequiredProperties(builderElement, avBuilderSetterNames);
-      case LOMBOK:
-        return getLombokRequiredProperties(builderElement);
-      default:
-        return Collections.emptyList();
-    }
   }
 
   // Keep a cache of these so that when declarationFromElement doesn't work,
   // we can still default correctly. Value is the property name to treat as
   // defaulted.
   private final Map<Element, String> defaultedElements = new HashMap<>();
```

Comment:
Let's just move this field to `LombokSupport`, since that is the only place it is used